py/packages/corpora_commander/api/llm_utils.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict
+
+from corpora_ai.llm_interface import LLMBaseInterface
+from corpora_ai.provider_loader import load_llm_provider
+
+
+def build_llm(provider: str, config: Dict[str, Any]) -> LLMBaseInterface:
```

Comment:
this is an ugly shim between our API and `load_llm_provider` ... needs a better abstraction for the client to be able to specify the model/provider and the endpoint load it dynamically .. .could have some types ... pretty meh implementation here but it's more fun to use the software, build books, and improve the software than make types for providers ATM.

Reply:
#78